src/Resources/Resources.js

```diff
@@ -160,7 +160,7 @@ export default class Resources extends Tool {
       key: name,
       val: value,
     }))
-
+    
```

Comment:
Try to avoid whitespace changes as they bloat the diffs with just cosmetic changes that do not have anything to do with fixes, additions of new features etc.
It should be easy to revert those in another commit to update & improve the pull request.